apisix/ssl/router/radixtree_sni.lua

```diff
@@ -67,12 +67,18 @@ local function decrypt_priv_pkey(iv, key)
         return key
     end
 
-    local decrypted = iv:decrypt(ngx_decode_base64(key))
-    if decrypted then
-        return decrypted
+    local decoded_key = ngx_decode_base64(key)
+    if not decoded_key then
+        core.log.error("base64 decode ssl key failed. key[", key, "] ")
```

Comment:
Better to mention that we will skip the bad key in the error message.

Reply:
added.